src/contexts/utils/getCurrencyFormatter.ts

```diff
@@ -44,3 +44,13 @@ const modifyFractionNumber = (amount: number) => {
   }
   return amount;
 };
+
+const parseScientificNotation = (amount: string) => {
+  if (amount.toString().includes('e') && amount.toString().includes('-')) {
+    return ['0', '00'];
+  }
+  if (amount.toString().includes('e') && amount.toString().includes('+')) {
+    return ['Infinity', 'Infinity'];
+  }
+  return amount.toString().split('.');
+};
```

Comment:
Can you please add some comments and tests to this to explain what the goal is here and why is it okay to round to 0.00 and Inifinity.Infinity?